rfcs/text/0000-alternative-planning.md

```diff
@@ -0,0 +1,315 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Roadmaps and Planning - an alternative perspective
+
+## Summary
+
+This informational RFC describes issues from the author's point of view with the Roadmaps and Planning RFC when it comes to software development
+in the the DCP. It then proposes alternative approaches to the problems that this RFC is looking to tackle.
+
+Please note: the list of reviewers chosen is somewhat arbitrary based on my memory of who concerns themselves with this
+stuff. And Github limits the number to 15. Apologies if I left you out - guaranteed I'd like you to read this even/especially 
+if you strongly disagree.
+
+## Author(s)
+
+[Justin Clark-Casey](mailto:justincc@ebi.ac.uk)
+
+## Shepherd
+***Leave this blank.*** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Roadmaps and planning are the crankshaft of the DCP. If we don't create a crankshaft suitable for the engine of development 
+then the DCP will run inefficiently or not at all. I'm concerned that the proposed process for creating these roadmaps and plans, as 
+described in the Roadmaps and Planning RFC and as an evolution of the current process, will not fit DCP organizational and development realities.
+
+This RFC tries to concisely describe those problems and propose alternative solutions. It comes as a result of being
+asked to write these alternatives up. Due to time constraints it's 
+nowhere near as comprehensive, well-written or referenced as I would like. Nonetheless, I hope it can help spur 
+constructive discussion.
+
+This RFC is the author's own point of view as a technical lead and someone with long experience in highly distributed 
+open-source projects that require co-operation between peer organizations and individuals.
+
+## Detailed Design
+
+From my understanding, the roadmaps and planning RFC describes a 70 day process where scientists, project leads, product owners, user experience
```

Comment:
RE _70 day process_ - as I slacked you on August 22 following the last DCP PM meeting _The PL(s) must deliver a draft annual roadmap by August 30 which will not be based on the RFC process. Using this roadmap as a “bootstrap”, I proposed to eliminate the annual roadmap process and focus on quarterly updates/refinements to the original roadmap in time for Release planning each quarter - similar to the Technical Objectives RFC. There were no objections, so I will be rewriting that section with Cara’s review.._ I expect to complete those updates this week, now that the Planning+Execution sections of the RFC have addressed pending comments.

Reply:
Apologies. I was punch-drunk on getting this out. I will take that into account soon.